src/main/java/gui/JStatGuiApplication.java

```diff
@@ -0,0 +1,43 @@
+package gui;
+
+import detail.JStatGuiGlobalData;
+import detail.MapDataContainer;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.concurrent.Executors;
+
+
+/**
+ * We ar running the application from an executable JAR. Thus,  it’s important to
+ * have a main class that will be executed when that JAR file is run. We also need at
+ * least a minimal amount of Spring configuration to bootstrap the application. 
+ *
+ */
+@SpringBootApplication
+public class JStatGuiApplication {
+
+    public static void main(String[] args) {
+
+        // do the needed initializations
+
+        // global object holding the loaded data sets
+        JStatGuiGlobalData.dataSetContainer = new MapDataContainer();
```

Comment:
TableDataSet?

Reply:
Maybe yes, we need to think if this ```MapDataContainer``` is needed altogether

Reply:
ahhhh sorry

Reply:
This is a general container it holds the name of a loaded dataset and an instance of the type that represents the dataset....it is not the dataset per se

Reply:
I think that I mixed thing up with the `MapDataContainer` and the `MapDataSet` which doesn't exist. I agree we should reflect on whether we need the `MapDataContainer`.

Reply:
No we do need the ```MapDataContainer``` or sth similar definitely....

Reply:
ok!